Tools/prepare_crashlytics.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
```

Comment:
You can't remove Tools/prepare_crashlytics.sh without removing it from .travis.yml